### PR TITLE
i18n(zh-cn): update `api-reference.mdx`

### PR DESCRIPTION
**Pull request by liruifengv**
#### Description (required)

 update `api-reference.mdx`

 Please merge https://github.com/withastro/docs/pull/9290 and https://github.com/withastro/docs/pull/9231 after this PR is merged.

#### Related issues & labels (optional)

- https://github.com/withastro/docs/pull/9224
- https://github.com/withastro/docs/pull/9261
- https://github.com/withastro/docs/pull/9254
- https://github.com/withastro/docs/pull/9174

### CODE REVIEW

**Comment by liruifengv** [comment]
Broken link will be fix after https://github.com/withastro/docs/pull/9292 is merged

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 1]
```diff
@@ -2075,6 +2137,468 @@ export const onRequest = defineMiddleware(async (context, next) => {
...
+#### `handler()` 属性
+
+<p>
+
+**类型：**`(input, context) => any`
+</p>
+
+`defineAction()` 函数接受一个 `handler()` 函数，其中包含在调用 Action 时要运行的服务器逻辑。此函数可以返回数据，这些数据将自动序列化并发送给调用者。
+
+`handler()` 函数接受用户输入作为其第一个参数。如果设置了 [`input`](#input-验证器) 验证器，则在将用户输入传递给 `handler()` 之前，将对其进行验证。第二个参数是一个 `context` 对象，包含大多数 Astro 的 [标准端点上下文](#端点上下文)，不包括 `getActionResult()`, `callAction()`, 和 `redirect()`。
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/api-reference.mdx:2522]:
```suggestion
`handler()` 函数被调用时，接受用户输入作为其第一个参数。如果设置了 [`input`](#input-验证器) 验证器，那么用户输入将在传递给 `handler()` 前进行验证。第二个参数是一个 `context` 对象，包含大多数 Astro 的 [标准端点上下文](#端点上下文)，不包括 `getActionResult()`, `callAction()`, 和 `redirect()`。
```

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 2]
```diff
@@ -2075,6 +2137,468 @@ export const onRequest = defineMiddleware(async (context, next) => {
...
+
+返回值被 [devalue 库](https://github.com/Rich-Harris/devalue) 解析。这支持 JSON 值，以及 `Date()`, `Map()`, `Set()`, 或 `URL()` 的实例。
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/api-reference.mdx:2524]:
```suggestion
返回值使用 [devalue 库](https://github.com/Rich-Harris/devalue) 进行解析。它支持 JSON 值，以及 `Date()`, `Map()`, `Set()`, 或 `URL()` 的实例。
```

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 3]
```diff
@@ -2075,6 +2137,468 @@ export const onRequest = defineMiddleware(async (context, next) => {
...
+#### `input` 验证器
+
+<p>
+
+**类型：**`ZodObject | undefined`
+</p>
+
+可选的 `input` 属性接受一个 Zod 验证器，用于在运行时验证处理程序的输入。如果 action 验证失败，将返回 [`BAD_REQUEST` 错误](#actionerror) 并跳过 `handler`。
+
+如果使用 `accept: 'form'`，则 `input` 必须使用 `z.object()` 验证器。
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/api-reference.mdx:2535]:
```suggestion
如果与 `accept: 'form'` 一起使用，则 `input` 必须使用 `z.object()` 验证器。
```

File: src/content/docs/zh-cn/reference/api-reference.mdx [hunk 4]
```diff
@@ -773,6 +773,46 @@ const orders = Array.from(Astro.locals.orders.entries());
...
+</p>
+
+`Astro.getActionResult()` 是一个返回 [Action](/zh-cn/guides/actions/) 提交结果的函数。它接受一个 Action 函数作为参数（例如 `actions.logout`），当接收到提交时返回一个 `data` 或 `error` 对象。否则，它将返回 `undefined`。
+
+
+```astro title="src/pages/index.astro"
+---
+import { actions } from 'astro:actions';
+const result = Astro.getActionResult(actions.logout);
+---
```
Comment (Nin3lee) [inline src/content/docs/zh-cn/reference/api-reference.mdx:791]:
```suggestion
---
import { actions } from 'astro:actions';

const result = Astro.getActionResult(actions.logout);
---
```

**Review by Nin3lee — CHANGES_REQUESTED** [review]

**Review by Nin3lee — APPROVED** [review]
LGTM! Thanks for ruifeng's such a huge update！

**Pull request merged**